kairon/api/app/routers/bot/bot.py

```
@@ -1700,3 +1703,91 @@
+    """
+    collection_id = DataProcessor.save_crud_collection_data(
+        payload=payload.dict(),
+        user=current_user.get_user(),
+        bot=current_user.get_bot()
+    )
+    return Response(message="Record saved!", data={"id": collection_id})
+
+
+@router.put("/collections/data/update", response_model=Response)
+async def update_collection_data(
+    payload: UpdateCollectionDataPayload = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)